lib/stream_rails/sync_policies.rb

```diff
@@ -5,7 +5,20 @@ module SyncPolicy
     module SyncCreate
 
       def self.included(base)
-        base.after_commit :add_to_feed, on: :create
+        if base.respond_to? :after_commit
+          base.after_commit :add_to_feed, on: :create
+        else
+          base.instance_eval do
+            define_method(:add_to_feed) {
```

Comment:
@tbarbugli any ideas on how I could avoid the redundancy. The `add_to_feed` method is already defined as a private method in the module.

Sequel doesn't seem to support `after_commit` hooks like AR. I currently have to overwrite the `after_create` and `before_destroy` hooks in Sequel and call the `add_to_feed` and `remove_from_feed` methods in there which is a huge PITA.

Any suggestions?